test/helpers/addAppRoutes.js

```diff
@@ -48,21 +48,16 @@ module.exports = (app, config) => {
   // Verify that API is working
   app.get('/withapi/verifyapi', async (req, res) => {
     // Make sure we have the correct session param
-    if (
-      !req.session
-      || !req.session.accessToken
-      || !req.session.refreshToken
-      || !req.session.accessTokenExpiry
-    ) {
+    if (!req.accessToken) {
       return res.json({
         success: false,
-        message: `Required: accessToken, refreshToken, and accessTokenExpiry in session. Instead, seassion was: ${req.session}`,
+        message: `Required: accessToken. Instead, seassion was: ${req.accessToken}`,
```

Comment:
typo here with "seassion". also maybe not the correct message anymore.

Reply:
Thank you! Should be fixed.